Client/game_sa/CGameSA.h

```diff
@@ -347,6 +364,7 @@ class CGameSA : public CGame
     bool         m_areWaterCreaturesEnabled{true};
     bool         m_isBurnFlippedCarsEnabled{true};
     bool         m_isFireballDestructEnabled{true};
+    bool         m_isExtendedWaterCannonsEnabled;
```

Comment:
```suggestion
    bool         m_isExtendedWaterCannonsEnabled{false};
```
You were using the default initializer, which uses a random value at whatever address this member ends up at. In a Debug build this is probably set to `false` anyway, so you would never notice this.